django_socio_grpc/tests/test_protobuf_registration.py

```diff
@@ -43,10 +43,12 @@ class Meta:
         fields = "__all__"
 
 
+class MyIntField(serializers.IntegerField):
+    ...
+
+
 class MySerializer(proto_serializers.ProtoSerializer):
-    user_name = serializers.IntegerField(
-        help_text=ProtoComment(["@test=comment1", "@test2=comment2"])
-    )
+    user_name = MyIntField(help_text=ProtoComment(["@test=comment1", "@test2=comment2"]))
```

Comment:
I didn't understand the issue on this one

Reply:
It's to test on a field subclass

Reply:
can you add a comment somewhere on why it can be a problem ?

Reply:
https://github.com/socotecio/django-socio-grpc/blob/master/django_socio_grpc/protobuf/proto_classes.py#L665